libs/blocks/locui/loc/index.js

```diff
@@ -17,7 +17,7 @@ import login from '../../../tools/sharepoint/login.js';
 import { getServiceUpdates } from '../utils/miloc.js';
 import { connectSK } from '../../../utils/sidekick.js';
 
-const LANG_ACTIONS = ['Translate', 'English Copy', 'Rollout'];
+const LANG_ACTIONS = ['Translate', 'English Copy', 'Rollout', 'Transcreate'];
```

Comment:
This is a bummer these are tightly coupled and we cannot get this data from the excel file easily.

Reply:
agreed